src/HospitalAPI/Controllers/Public/AllergenController.cs

```diff
@@ -12,7 +12,6 @@ namespace HospitalAPI.Controllers.Public
 {
     [Route("api/public/allergen")]
     [ApiController]
-    [Authorize(Roles="Patient")]
     public class AllergenController : ControllerBase
```

Comment:
Add authorization on method Create allergen, not every role can create allergen.